systemtests/scripts/cleanup

```diff
@@ -24,4 +24,10 @@ rm -rf /tmp/chrome-user-data-*
 
 ${scripts}/drop_bareos_database ${DBTYPE} >/dev/null 2>&1
 
+# extend local systemtests
+localsetup="testrunner.d/$(basename $0)"
+echo "debug localsetup is ${localsetup}"
+if [ -f ${localsetup} ];then
+  ${localsetup}
+fi
```

Comment:
if the custom cleanup runs last in `cleanup` then the setup should run first in `setup` (and not `start_bareos.sh`).  Startup & Cleanup should be like brackets; if you have multiple it should look like this
`{[]}` and not `{[}]`